pipes/WDL/workflows/read_depths.wdl

```diff
@@ -0,0 +1,34 @@
+version 1.0
+
+import "../tasks/tasks_read_utils.wdl" as read_utils
+
+workflow read_depths {
+    meta {
+        description: "Generates read depth tables."
+        author: "Broad Viral Genomics"
+        email:  "viral-ngs@broadinstitute.org"
+    }
+
+    parameter_meta {
+        aligned_bams: {
+            description: "Reads aligned to a reference sequence.",
+            patterns: ["*.bam"]
+        }
+    }
+
+    input {
+        Array[File]+ aligned_bams
+    }
+
+    scatter(aligned_bam in aligned_bams) {
```

Comment:
Any reason this is scattered instead of just making it a one-sample workflow (and using terra's ability to launch a one-sample wdl on a whole table of inputs)

Reply:
To mirror the isnvs_lofreq workflow, which also takes in an array of bam files (aligned_bams). Perhaps it is helpful if you have multiple bams per sample and you would like each sample to have one data table row.

Reply:
Right, if you want an `Array[File]` of read depths per sample, this seems like the right approach